sdk/eventhubs/azure_messaging_eventhubs/src/event_processor/load_balancer.rs

```diff
@@ -149,6 +149,7 @@ impl LoadBalancer {
 
         let minimum_required = partition_ids.len() / grouped_by_owner.len();
         let mut maximum_allowed = minimum_required;
+        #[expect(manual_is_multiple_of)]
```

Comment:
This is a clippy lint, and don't use `expect` because the lint is new and won't fire on older compilers.
```suggestion
        #[allow(clippy::manual_is_multiple_of)]
```